read-fonts/Cargo.toml

```diff
@@ -10,9 +10,13 @@ categories = ["text-processing", "parsing", "graphics"]
 
 [features]
 std = ["font-types/std"]
-test_data = []
+test_data = ["font-test-data"]
 traversal = ["std"]
 default = ["traversal"]
 
 [dependencies]
 font-types = { version = "0.1.4", path = "../font-types" }
+font-test-data = { path = "../font-test-data", optional = true }
```

Comment:
I'm concerned that this may also break packaging if we aren't publishing this crate, so hold on merging until I can play around a bit more and confirm that.